contracts/eigenlayer/script/WavsMirrorDeployer.s.sol

```diff
@@ -41,11 +50,11 @@ contract WavsMirrorDeployer is Script, IECDSAStakeRegistryTypes {
         // initialize the operator set
         console2.log("Configuring initial state...");
         WavsMirrorDeploymentLib.setInitialConfiguration(deployment, configuration);
-
-        // deploy the handlers
-        console2.log("Deploying ServiceHandlers as admin...");
-        deployment = WavsMirrorDeploymentLib.deployServiceHandlers(deployment);
-
+        if (deploymentStage == DEPLOYMENT_STAGE_TESTNET) {
```

Comment:
Okay, I see the use of reusing the scripts here. But I don't like this variable. I propose two alternate solutions which I am fine with both.

1. Add a boolean env var `WAVS_MIRROR_DEPLOY_HANDLERS` or such and use that to gate the service handler deployment stuff (same logic you have now, just different env variable handling)
2. Copy this deploy script and make a completely different one for mock deploy.

I think the first one might be easier now, but ideally the real deploy becomes a multi-chain aware script later (rather than the intermediate json file) and these two approaches will deviate and no longer fit in one script file.